nautobot/dcim/tables/template_code.py

```diff
@@ -8,6 +8,28 @@
 {% endif %}
 """
 
+PATHENDPOINT = """
+{% if value.destination %}
+    <a href="{{ value.destination.parent.get_absolute_url }}">{{ value.destination.parent }}</a>
+    <i class="mdi mdi-chevron-right"></i>
+    <a href="{{ value.destination.get_absolute_url }}">{{ value.destination }}</a>
+    {% with traced_path=value.origin.trace %}
+        {% for near_end, cable, far_end in traced_path %}
+            {% if near_end.circuit %}
+                <small>via<i class="fa fa-fw fa-globe" title="Circuit"></i>
```

Comment:
The globe icon isn't rendering because Nautobot (like NetBox 2.10) doesn't use FontAwesome (`fa`), but instead uses Material Design Icons (`mdi`).

Reply:
I don't think the globe is needed, I just was coping some code that I found referenced.  I am not so great at classes, should I just remove this or is there something you would suggest from `mdi`

Reply:
I'm fine with removing it.

Reply:
Thanks! I removed this.  We still have the via, so it should match up with the screenshot.